src/platform/user/profile/vet360/components/base/TransactionPending.jsx

```diff
@@ -1,9 +1,10 @@
 import React from 'react';
+import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 
-import environment from 'platform/utilities/environment';
+import { profileShowNotifications } from '../../selectors';
```

Comment:
Could you change this from `'../../selectors'` to `'vet360/selectors'`? This is optional but I find it a little bit clearer and easier to understand vs. trying to figure out which directory is two directories up.